synapse/lib/cell.py

```diff
@@ -375,11 +375,9 @@ async def issue(self, nexsiden: str, event: str, args, kwargs, meta=None):
         return path
         '''
         try:
-            await self.cell.nexsroot.issue(nexsiden, event, args, kwargs, meta)
+            return await self.cell.nexsroot.issue(nexsiden, event, args, kwargs, meta)
         except asyncio.CancelledError:  # pragma: no cover  TODO:  remove once >= py 3.8 only
             raise
-        except Exception:
```

Comment:
If this is no longer eating exceptions then you need to update the docstring accordingly.